cucumber/authenticators_jwt/features/support/authn_jwt_helper.rb

```diff
@@ -1,31 +1,45 @@
 # frozen_string_literal: true
 
 # Utility methods for JWT authenticator
+require 'cucumber/authenticators_oidc/features/support/authn_oidc_helper'
 
 module AuthnJwtHelper
```

Comment:
@nessiLahav  could you please add a comment to this module? (code climate issue)